packages/webapp/pages/account/profile.tsx

```diff
@@ -161,14 +161,14 @@ const AccountProfilePage = (): ReactElement => {
             <ImageInput
               id={coverId}
               className={{
-                root: 'w-full',
+                root: 'w-full max-w-[19.25rem]',
                 container:
                   'border-0 bg-background-subtle hover:bg-accent-pepper-subtlest',
                 img: 'object-cover',
               }}
               size="cover"
               initialValue={currentCoverImage}
-              fallbackImage={undefined}
```

Comment:
OMG 🤦

Reply:
The usage of `null` is kinda tricky though. So it is really easy to miss. I wonder if there is a better approach to these kind of cases.